app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+class TweetsController < ApplicationController
+
+  get "/tweets" do
+    @tweets = Tweet.all
+    if logged_in?
+      erb :"tweets/tweets"
+    else
+    redirect to "/login"
+    end
+  end
+
+
+  get "/tweets/new" do
+    if logged_in?
+    erb :"tweets/create_tweet"
+  else
+    redirect to "/login"
+  end
+end
+
+  post "/tweets" do
+    if !params[:content].empty?
+      @tweet = Tweet.create(content: params[:content])
+    # binding.pry
+      @tweet.user_id = session[:user_id]
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```